stripe:
    binarizer:
        s_min: 100
        h_min: 2
        h_max: 30
        # v_min: 150

        # h_min: 0
        # h_max: 60


    enable_col_cor: true

    median_blur:
        ksize: 5

    equalizer:
        unused: 0.01
        channels_count: 0
        # channels_count: 3

    stripes:
        min_length: 60
        min_width: 10

    approx_dist: 3.0

gate:
    red_stripe:
        binarizer:
            h_min: -20
            h_max: 50

        median_blur:
            ksize: 5

        equalizer:
            unused: 0.01

        enable_col_cor: 1
        stripes:
            sides_ratio: 1.5
            approx_diff: 3.0
            # min_length: 5
            max_width: 10

    green_stripe:
        binarizer:
            h_min: 60
            h_max: 85
            s_min: 125
            s_max: 160
            v_min: 100
            v_max: 150

        median_blur:
            ksize: 7

        equalizer:
            unused: 0.01
        stripes:
            sides_ratio: 1.5
            approx_diff: 3.0
            # min_length: 5
            max_width: 20

fargate:
    median_big:
        ksize: 17
        # ksize: 47
    median:
        ksize: 5

    dilate:
        side_: 7

    green:
        h_min: 60
        h_max: 80

    hsv:
        # h_max: 5
        s_max: 70
        v_min: 240

    approx_dist: 1

    unused: 0.01
    channels_count: 0
    # channels_count: 3

    dx: 1
    most_common_part: 0.02
    min_gate_width: 50
    cell_pixels: 2
    # cell_pixels: 1
    hough_thresh: 0.11

circle:
    green:
        h_min: 50
        h_max: 80
        s_min: 50
        # v_min: 100

    median:
        ksize: 11

    other:
        h_min: -5
        h_max: 50
        s_min: 130
        v_min: 100

    approx_dist: 1
    circles:
        min_sides_ratio: 0.5
        min_width: 20
        max_length: 400

    unused: 0.01
    channels_count: 0

bin:
    enable_col_cor: true
    orange:
        binarizer:
            s_min: 100
            h_min: 2
            h_max: 30
            # v_min: 150

        enable_col_cor: true

        median_blur:
            ksize: 5

        equalizer:
            unused: 0.01
            count: 0

        stripes:
            min_length: 40
            min_width: 10

        approx_dist: 3.0

    white:
        binarizer:
            s_max: 20
            v_min: 150

        enable_col_cor: true

        median_blur:
            ksize: 5

        equalizer:
            unused: 0.01
            count: 0

        stripes:
            min_length: 40
            min_width: 10

        approx_dist: 3.0

channel:
    border_delta: 20

    binarizer:
        h_min: 20
        h_max: 50

    median_blur:
        ksize: 5

    equalizer:
        unused: 0.01
        channels_count: 0

    enable_correction: false

    stripes:
        min_length: 10

    approx_dist: 3.0
